lib/mixlib/shellout/windows/core_ext.rb

```diff
@@ -349,14 +349,12 @@ def create(args)
         procinfo[:hThread] = 0
       end
 
-      process = ProcessInfo.new(
+      ProcessInfo.new(
         procinfo[:hProcess],
         procinfo[:hThread],
         procinfo[:dwProcessId],
         procinfo[:dwThreadId]
       )
-
-      [ process, profile, token ]
     end
```

Comment:
Can you just rename `def create` here to `def create2` and then construct:

```
def create(args)
  create2(args)[0]
end
```

Then above call:  `process, profile, token = Process.create2(create_process_args)`